collector-backend/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "collector-backend"
-version = "1.1.3"
+version = "1.1.4"
```

Comment:
Why is this bumped?

Reply:
Because some dependencies were updated to address some security advisories on them.